src/sp/it/pl/util/action/ActionManager.kt

```diff
@@ -101,7 +101,7 @@ object ActionManager {
     fun startActionListening() {
         if (isActionListening) throw IllegalStateException("Action listening already running")
         startLocalListening()
-        if (isGlobalShortcutsSupported && globalShortcuts.get()) startGlobalListening()
```

Comment:
This has good intention, but it is currently impossible to prevent user changing the value and bypass the isSupported check. Thus it must stay as is.

Reply:
Hm or perhaps not, will see after adding my changes on top.